recipe/meta.yaml

```diff
@@ -22,7 +22,7 @@ requirements:
     - python >=3.8
     - requests >=2.20.0
     - globus-sdk >=3.14.0,<4
-    - globus-compute-common =0.2.0
```

Comment:
We don't do pre-releases without a special label. This package will have to wait until it has a release that depends on a stable version of globus-compute-common

https://conda-forge.org/docs/maintainer/knowledge_base.html#creating-a-pre-release-build